src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -587,7 +592,40 @@ bool clang_c_convertert::get_function(const clang::FunctionDecl &fd, exprt &)
   return false;
 }
 
+bool clang_c_convertert::get_function_body(
+  const clang::FunctionDecl &fd,
+  exprt &new_expr)
+{
+  assert(fd.hasBody());
+
+  exprt body_exprt;
+  if(get_expr(*fd.getBody(), body_exprt))
```

Comment:
Which case do we return true?

Reply:
Done. 
Added a comment to explain a failed case